salt/_pillar/metalk8s_endpoints.py

```diff
@@ -16,6 +16,48 @@ def __virtual__():
         return __virtualname__
 
 
+def namespaced_service_endpoints_nodeport(service, namespace, kubeconfig):
```

Comment:
ditto, I'm not sure that it's useful to add the `namespaced` prefix: eveything seems to be namespaced in k8s anyway.